frontend/src/components/Header/Navbar/MobileNavbar.js

```diff
@@ -0,0 +1,57 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faBars, faX } from "@fortawesome/free-solid-svg-icons"
+import { useEffect, useRef, useState } from 'react'
+import NavbarLink from './NavbarLink'
+import SearchBar from '../TitleBanner/SearchBar'
+
+const MobileNavbar = () => {
+  const hamburger = <FontAwesomeIcon icon={faBars} className="text-2xl" />
+  const xMark = <FontAwesomeIcon icon={faX} className="text-2xl text-indigo-400" />
+  const [show, setShow] = useState(false)
+  const ref = useRef()
+
+  const handleToggle = () => {
+    setShow(!show)
+  }
+
+  const triggerClose = () => {
+    setShow(false)
+  }
+
+  useEffect(() => {
+    const triggerClose = e => {
```

Comment:
You have two consts with the same name. The useEffect one should probably be named something different to avoid confusion